### ISSUE TITLE
we have PAX issue at a hardened Gentoo

### ISSUE THREAD

**Issue by toralf** [unittest-cpp/unittest-cpp#104]
The origin is here : https://bugs.gentoo.org/show_bug.cgi?id=576806
Any hints ?

**Comment by pjohnmeyer** [comment]
Having looked at the issue, the only thing immediately jumping out at me would be in `TestTest.cpp`. The test `CrashingTestsAreReportedAsFailures` intentionally calls a function at address 0x0:

```cpp
virtual void RunImpl() const
{
   reinterpret_cast< void (*)() >(0)();
}
```

If you or somebody can verify for me that this is indeed the cause, then it will be easy enough to come up with a fix.

**Comment by pjohnmeyer** [comment]
@toralf I consider this issue addressed by #115, although there was some disagreement on my implementation there. Do you consider that to have fixed the issue, once a release is made?

**Comment by toralf** [comment]
well, dev-libs/unittest++-1.6.1 builds fine here since sbraz already cherry-picked a patch, so this can be closed for now IMO.
